lib/core/network/protocols/http+websocket.js

```diff
@@ -0,0 +1,910 @@
+/*
```

Comment:
Wot, I don't like the use of characters like `+` in file naming, kindda weird.
Also, I believe this goes against node export naming convention, shouldn't the file name reflect the name of the class defined within?

Reply:
You're no fun: I do like esoteric characters in file names, messing up with regexes when looking for them. :expressionless: